crowbar_framework/app/models/api/node.rb

```diff
@@ -16,16 +16,20 @@
 
 module Api
   class Node < Tableless
+    attr_reader :node_name
```

Comment:
I did this before but then I realized there will be lot of methods which will take a name so I decided to name the accessor more explicitly.
But yes could be named also simply 'name'